db/migrate/20180827194804_add_names_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddNamesToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string
```

Comment:
you can specify the not null here with
`add_column :users, :first_name, :string, null: false`
and avoid creating a separate migration